charts/operator-wandb/charts/flat-run-fields-updater/templates/hpa.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.autoscaling.enabled }}
+---
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ include "flat-run-fields-updater.fullname" . }}
+  labels:
+    {{- include "flat-run-fields-updater.labels" . | nindent 4 }}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: {{ include "flat-run-fields-updater.fullname" . }}
+  minReplicas: {{ .Values.autoscaling.minReplicas }}
+  maxReplicas: {{ .Values.autoscaling.maxReplicas }}
+  metrics:
+    {{- if .Values.autoscaling.targetCPUUtilizationPercentage }}
+    - type: Resource
+      resource:
+        name: cpu
+        target:
+          type: Utilization
+          averageUtilization: {{ .Values.autoscaling.targetCPUUtilizationPercentage }}
+    {{- end }}
+    {{- if .Values.autoscaling.targetMemoryUtilizationPercentage }}
+    - type: Resource
+      resource:
+        name: memory
+        target:
+          type: Utilization
+          averageUtilization: {{ .Values.autoscaling.targetMemoryUtilizationPercentage }}
+    {{- end }}
```

Comment:
Let's start by defining these statically for simplicity's sake. We can always add templating later if the need arises. This also prevents customers from accidentally modifying the values and it follows the pattern we are currently use in the codebase

Reply:
I don't think statically defining these is a good approach for a couple of reasons.  First, this is the helm recommended file/format for an hpa and already has everything defined for us out of the box so we can hit the ground running when we're ready for hpa architecture.  Second, until we actually start using the system we're not going to have any real insight for what these values should be.  Leaving these as dynamic placeholders means we don't have to do the extra work of figuring out baselines before merging this in.